ballerina-tests/ai.np-tests/main.bal

```diff
@@ -25,6 +25,10 @@ function queryAboutCountry(string query) returns string|error => natural (mp) {
     Which country is ${query}?
 };
 
+function queryACalculation() returns string|error => natural (mp) {
```

Comment:
Is this test correct? This works already, right? The issue is when there is a type reference type.

```ballerina
type Error distinct error;

function queryACalculation() returns error? {
    string|Error x = check natural (mp) {
        What is 1 + 2 ?
    };
}
```